concepts/strings/about.md

```diff
@@ -0,0 +1,220 @@
+# Introduction
+
+Characters and Strings form the basis for text representation in Elm.
+Strings can be thought of as lists of characters and most `List` functions have their equivalent in the `String` module.
+
+## Characters
+
+The `Char` type represents a single unicode character with single quotes e.g. 'a'. 
+The `Char` module provides predicate functions (`Char -> Bool`) to test qualities of a character: `isUpper`, `isLower`, `isAlpha`, and `isAlphaNum`
+The module also provides functions to convert a character `toUpper`, `toLower`, `toLocaleUpper`, `toLocaleLower`, as well as to/from their numerical unicode value (`Int`) with `toCode` and `fromCode`.
+
+## Strings
+
+The `String` type provides a built-in representation for efficient string manipulation and can represent any sequence of unicode characters.
+Strings are represented with double quotes: `"this is a string"`.
+Multi-line strings are represented with triple double quotes and can have unescaped quotes and newlines.
+
+```elm 
+haiku = 
+"""
+    Code's strings entwine
+    , In binary, thoughts combine
+    , Logic's elegant line.
+"""
+```
+
+## String Manipulation
+
+`reverse : String -> String` reverses the string.
```

Comment:
There is a convention in the repo to use `-->` to indicate the outcome of a function in the documentation like this.
To be fair, I don't think it is documented anywhere.

So maybe something like this:

Strings are reversed using `String.reverse`
```
reverse "Hello world." --> ".dlrow olleH"
```

You could have a look at one of the other concepts and try and follow the pattern. I think there is a pattern to the other concepts, but also, it probably isn't 100% followed, so let me know if you have a strong objection to it.

Reply:
No problem. I changed all the `==` to `-->' indented on the following line. Other concepts in this track vary between inline and indented on the next line. I think the latter improves readability and is more in keeping with Elm format rules.